src/Tar/project.json

```diff
@@ -1,16 +1,21 @@
 ﻿{
   "version": "0.1.0",
   "frameworks": {
+    "netstandard1.6": {
+      "dependencies": {
+        "System.IO": "4.1.0",
+        "System.IO.FileSystem": "4.0.1",
+        "System.Linq": "4.1.0",
+        "System.Runtime": "4.1.0",
+        "System.Runtime.Extensions": "4.1.0",
+        "System.Text.Encoding": "4.0.11",
+        "System.Text.Encoding.Extensions": "4.0.11",
+        "System.Threading.Tasks": "4.0.11"
+      }
```

Comment:
@jterry75 This works, but I wonder if there is a better way for me to express these dependencies.  Any ideas?